infinit/github-repo

```diff
@@ -0,0 +1 @@
+http://github.com/infinit/infinit-docker
```

Comment:
Can you swap this to use `https` please? (it'll redirect, but better to be explicit IMO)